config/controller/admin/config.config.php

```diff
@@ -55,7 +55,7 @@
                     ),
                 ),
             ),
-            99 => array(
+            2 => array(
```

Comment:
I yes your maybe true ! : but why the before key (https://github.com/novius/bru_google_seo_tools/pull/4/files#diff-cba3c1bfda1f498c5955313ee026df53R10) => `1` will not be overwrite also by another website ?